### PR TITLE
Feature: Examples

### PR DESCRIPTION
**Pull request by marcbouchenoire**
This PR has various goals related to upcoming detail pages on https://liveblocks.io/examples.

- Centralize all examples in the public repo
	- [x] Add missing examples
	- [x] Harmonize all examples
- Simplify code structure and reduce boilerplate
	- [x] Remove duplicate files when possible
	- [x] Reduce non-example code (errors, loading...)
- Publish all examples individually
	- [x] Add logic to set public keys and room IDs via query parameters when needed

### CODE REVIEW

**Review by GuillaumeSalles — APPROVED** [review]
Incredible work! So tedious and more complex than it seems 👏 

Thank you so much for doing that @marcbouchenoire !

Did you make a guideline doc for all the rules that you applied to all examples? For example, 
- `prettier` should be in every project in `devDependencies`
- PUBLIC_KEY should be overridable from query
- etc, etc

I think it would helpful for future contributors and it will strengthen your fondation.

**Comment by marcbouchenoire** [comment]
> Did you make a guideline doc for all the rules that you applied to all examples?
>
> I think it would helpful for future contributors and it will strengthen your fondation.

@GuillaumeSalles I only had a few notes but yeah fully agree, there's a lot of info/knowledge that could be lost if I keep them all to myself. I started to gather them in a Notion doc and will try to make this as complete and helpful as possible over time. https://www.notion.so/liveblocks/Examples-3afe2a961a9d455787c6a685964af5c3

File: examples/express-javascript-live-cursors/index.js [hunk 1]
```diff
@@ -1,25 +1,38 @@
 const express = require("express");
```
[the remark this answers is not in the document]
Comment (marcbouchenoire) [reply examples/express-javascript-live-cursors/index.js:1]: We shouldn't have un-deployable examples so yeah this one might be a problem.

The front-end only version even already exists (https://github.com/liveblocks/liveblocks/tree/main/examples/javascript-live-cursors), so if you feel like this one can go I'll remove it.

File: examples/javascript-live-cursors/app.js [hunk 2]
```diff
@@ -1,12 +1,64 @@
 import { createClient } from "@liveblocks/client";
 
+const query = new URLSearchParams(window?.location?.search);
+
+/**
+ * Replace by your public key from https://liveblocks.io/dashboard/apikeys.
+ */
+let PUBLIC_KEY = "pk_YOUR_PUBLIC_KEY";
+
+/**
```
[the remark this answers is not in the document]
Comment (marcbouchenoire) [reply examples/javascript-live-cursors/app.js:10]: I'm with you, this was one point I wasn't too sure about the way I approached it. I'll nudge all current variants to something like your example.

File: examples/express-javascript-live-cursors/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (GuillaumeSalles) [reply examples/express-javascript-live-cursors/index.js:1]: So yeah, let's remove it 👍

**Pull request merged**